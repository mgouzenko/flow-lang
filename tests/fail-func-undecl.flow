// Should not be able to call a function that has not been declared

int double_sum(int a, int b) {
  int c = sum(a, b) + sum(a, b); // FAIL
  return c;
}

int sum(int a, int b) {
  return a + b;
}

int main() {
  numGen(chan);
  sum(chan);
}