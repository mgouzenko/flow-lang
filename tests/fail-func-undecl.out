Fatal error: exception Failure("Undeclared function sum")